doc/lua_api.txt

```diff
@@ -3989,6 +3989,12 @@ Environment access
         * mode = `"quick"`: Clear objects immediately in loaded mapblocks,
                             clear objects in unloaded mapblocks only when the
                             mapblocks are next activated.
+* `minetest.load_area(pos1[, pos2])`
+    * Load the mapblock which contains `pos1`
+      or blocks in the area from `pos1` to `pos2` if `pos2` is specified.
+    * This function does not trigger map generation.
+      After executing it, `minetest.get_node_or_nil` returns the node even
```

Comment:
Line needs a '*'

Reply:
'After execution', as 'it' could be referring to 'minetest.get_node_or_nil'

Reply:
IMO this line is not required because it's the behaviour of `get_node*` to only return the actual node when the mapblock is loaded.

Reply:
I agree line is not needed.